app/upgrades/v16/upgrades.go

```diff
@@ -63,6 +66,12 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		updateTokenFactoryParams(ctx, keepers.TokenFactoryKeeper)
+
 		return migrations, nil
 	}
 }
+
+func updateTokenFactoryParams(ctx sdk.Context, tokenFactoryKeeper *tokenfactorykeeper.Keeper) {
+	tokenFactoryKeeper.SetParams(ctx, tokenfactorytypes.NewParams(nil, NewDenomCreationGasConsume))
```

Comment:
I assume this is purposely setting the fee to nil, but commenting in case the intention was to add gas-consume without modifying the fee.

Reply:
@nicolaslara Proposal 489 (https://www.mintscan.io/osmosis/proposals/489) states that the fee is to be set "to zero in the UpgradeHandler of the next chain upgrade".

I'm going to add a test for the gas consumption then this PR should be ready.